spring-integration-core/src/test/java/org/springframework/integration/aggregator/FluxAggregatorMessageHandlerTests.java

```diff
@@ -157,6 +158,7 @@ void testCustomCombineFunction() {
 	}
 
 	@Test
+	@Disabled("Time-sensitive")
```

Comment:
Please, consider to do something like this:
```
git checkout main
git pull upstream main
git checkout -
git rebase main
```

and then `push --force` to this PR.

I mean it might not be a single not related to your PR change, so we will suffer reviewing some extra code existing already in the `upstream/main`